packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/price/translations/Messages_pl_PL.json

```diff
@@ -0,0 +1,8 @@
+{
+  "price_ht_label": "{{price}} netto",
+  "price_ttc_label": "{{price}} brutto",
+  "price_free": "W ofercie",
+  "price_gst_excl_label": "{{price}} netto",
+  "price_gst_incl_label": "{{price}} brutto",
+  "price_per_month": "/m-c"
```

Comment:
Just a reminder that translations added shoud only be `fr_FR`

Reply:
Other language translations removed